app/controllers/admin/application_controller.rb

```diff
@@ -2,4 +2,5 @@ class Admin::ApplicationController < ApplicationController
   include Pundit
 
   before_action :authenticate_admin_or_organiser!
+
```

Comment:
Please remove this. Don't use empty lines around method, class, module, block bodies. [link][1]

[1]: https://github.com/bbatsov/ruby-style-guide#empty-lines-around-bodies